concepts/observability-strategy/strategy.md

```diff
@@ -98,15 +103,17 @@ All concepts follow general rules and will provide harmonized user APIs:
 - **Custom output protocols can be integrated**: Users can have custom transformation from the supported protocol, usually with additional operational effort (like running a dedicated Otel Collector sidecar or deployment in the users' Namespace). Furthermore, Kyma offers typical output plugins of the used collector technology as part of a pipeline definition; however, only with limited support.
 - **Blueprints are fully integrated**: Blueprints for local backend deployments are integrated with the used pipeline mechanism, so that they work instantly.
 
+## Conclusion
+
+Shifting the focus of Kymas Observability feature away from an end-to-end observability solution having major drawbacks (in-cluster, opinionated, not flexible enough, ..), into a solution which supports the user with the runtime centric tasks of instrumentation, collection and integration of telemetry data into external systems in a vendor-neutral way, which solves an actual problem of enterprise users - getting telemetry data reliable into an external system with few efforts.
```

Comment:
```suggestion
Shifting the focus of Kyma's Observability feature away from an end-to-end observability solution that has major drawbacks (in-cluster, opinionated, not flexible enough, ..), to a solution that supports the user with the runtime-centric tasks of instrumentation, collection, and integration of telemetry data into external systems in a vendor-neutral way, which solves an actual problem of enterprise users - getting telemetry data reliable into an external system with few efforts.
```

Even though this is a very very long sentence, it has no predicate. What do you want to say with this? We need to split it in 2-3 shorter sentences.

Reply:
Has been rephrased